test/make-images-push-to-local-registry.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 cd ..
-DOCKER_BUILDKIT=1 make docker-image DOCKER_IMAGE_TAG="$2"
+IGNORE_GIT_STATUS=1 DOCKER_BUILDKIT=1 make docker-image DOCKER_IMAGE_TAG="$2"
```

Comment:
But this should not be there. Rather add any offending temp files to `.gitignore`.